frontend/src/components/core/AppView/AppView.tsx

```diff
@@ -111,15 +116,22 @@ function AppView(props: AppViewProps): ReactElement {
   )
 
   const layout = wideMode ? "wide" : "narrow"
+  const hasElements = !elements.sidebar.isEmpty
```

Comment:
I'm not the biggest fan of the `hasElements` name, but I'll just leave coming up with something better as a TODO for now

Reply:
I think `hasElements` is fine. Alternatives: `containsElements`, `hasComponents`, ...